app/views/content/is-teaching-right-for-me/all-subjects.md

```diff
@@ -0,0 +1,299 @@
+
+## Business studies 
+
+As a business studies teacher you'll teach pupils more about the business world and how to apply that knowledge to contemporary business issues. 
+
+<a href="https://www.find-postgraduate-teacher-training.service.gov.uk/results?age_group=secondary&applications_open=true&can_sponsor_visa=false&has_vacancies=true&l=2&subjects%5B%5D=08&visa_status=false">Search for business studies teacher training courses</a>.
+
+## Chemistry
+
+<a href="https://getintoteaching.education.gov.uk/funding-and-support/scholarships-and-bursaries?funding_widget%5Bsubject%5D=chemistry&button=#funding-widget">Tax-free scholarships of £30,000 and bursaries of £28,000 are available for trainee chemistry teachers</a>.
+
+<a href="/is-teaching-right-for-me/chemistry">You can learn more about teaching chemistry</a>. 
+
+ <a href="https://www.find-postgraduate-teacher-training.service.gov.uk/results?age_group=secondary&applications_open=true&can_sponsor_visa=false&has_vacancies=true&l=2&subjects%5B%5D=F1&visa_status=false">Search for chemistry teacher training courses</a>.
+
+## Citizenship 
+
+As a citizenship teacher you'll teach pupils how to become informed and active citizens in society. You'll teach pupils the importance of democracy and politics.    
+
+ <a href="https://www.find-postgraduate-teacher-training.service.gov.uk/results?age_group=secondary&applications_open=true&can_sponsor_visa=false&has_vacancies=true&l=2&subjects%5B%5D=09&visa_status=false">Search for citizenship teacher training courses</a>.
+
+## Classics
+
+As a classics teacher you'll teach pupils about the literature and culture of the classical world, like the Roman empire, and how it relates to themes and issues of today's world.
+
+<a href="https://www.find-postgraduate-teacher-training.service.gov.uk/results?age_group=secondary&applications_open=true&can_sponsor_visa=false&has_vacancies=true&l=2&subjects%5B%5D=Q8&visa_status=false">Search for classics teacher training courses</a>.
+
+## Communication and media studies  
+
+As a communication and media studies teacher you can offer pupils insight into how the world communicates through a range of different media.  
+
+<a href="https://www.find-postgraduate-teacher-training.service.gov.uk/results?age_group=secondary&applications_open=true&can_sponsor_visa=false&has_vacancies=true&l=2&subjects%5B%5D=P3&visa_status=false">Search for communication and media studies teacher training courses</a>.
+
+## Computing
+
+<a href="https://getintoteaching.education.gov.uk/funding-and-support/scholarships-and-bursaries?funding_widget%5Bsubject%5D=computing&button=#funding-widget">Tax-free scholarships of £30,000 and bursaries of £28,000 are available for eligible trainee computing teachers</a>.
```

Comment:
@caitlin-reidy Realise we've got the new computing page but I still think we should have a line of intro for each of the subjects we have pages for. So they all follow the same format and are consistent. And also because people may not have read them and we want to get them excited about chemistry so they want to then read it.

Otherwise for this one we just have 3 links.

And same as previous comments I'd put bursary info 2nd.